Makefile

```diff
@@ -29,7 +29,7 @@ test:
 
 staticcheck:
 	@printf $(COLOR) "Run static check..."
-	@go install honnef.co/go/tools/cmd/staticcheck@2022.1.3
+	@go install honnef.co/go/tools/cmd/staticcheck@latest
```

Comment:
I think we should pin a version (even if it is the latest as of now). It can be confusing for contributors that that external project making a release may force them to make code alterations unrelated to their PR just to pass CI.